Dockerfile

```diff
@@ -1,15 +1,18 @@
-FROM node:18-bullseye-slim
+FROM node:20-slim
 WORKDIR /app
+EXPOSE 9000
```

Comment:
should probably change this to 3000 since that's the defacto default port everywhere